app/src/main/java/jp/kztproject/rewardedtodo/presentation/HomeActivity.kt

```diff
@@ -58,6 +61,24 @@ class HomeActivity : ComponentActivity() {
             ) { padding ->
                 RewardedTodoApp(padding, navController)
             }
+
+            if (showSettingDialog) {
+                var todoistAuthFinished by remember { mutableStateOf(false) }
+                val launcher = rememberLauncherForActivityResult(ActivityResultContracts.StartActivityForResult()) {
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/kseito/RewardedTodo/blob/427d130de07c1ddd0b808f8f2f01f5e9547df0f2/app/src/main/java/jp/kztproject/rewardedtodo/presentation/HomeActivity.kt#L67"></span>Argument should be on a separate line (unless all arguments can fit a single line)</td>
    </tr>
  </tbody>
</table>